frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -25,83 +22,105 @@ let updatedClusterJSON;
 let updatedCnt;
 let updatedPods;
 
-describe('Check add capacity functionality for ocs service', () => {
-  beforeAll(async () => {
-    // check whether the cluster is present or not, and check for its ready state
-    // if not present exit and console that no cluster is present, will be send in next PR
+const storageCluster = JSON.parse(
+  execSync(`kubectl get ${KIND} -n ${NAMESPACE} -o json`).toString(),
+);
 
-    const { name } = storageCluster.items[0].metadata;
-    clusterJSON = JSON.parse(
-      execSync(`kubectl get -o json -n ${namespace} ${kind} ${name}`).toString(),
-    );
-    previousCnt = _.get(clusterJSON, 'spec.storageDeviceSets[0].count', undefined);
-    const uid = _.get(clusterJSON, 'metadata.uid', undefined).toString();
-    previousPods = JSON.parse(execSync(`kubectl get pods -n ${namespace} -o json`).toString());
+describe('Check availability of OCS cluster', () => {
+  if (storageCluster.items[0]) {
+    it('Should check if the OCS cluster is Ready for expansion', () => {
+      expect(_.get(storageCluster.items[0], 'status.phase')).toBe('Ready');
+    });
+  } else {
+    it('Should state that OCS cluster is not ready for expansion', () => {
+      expect(storageCluster.items[0]).toBeUndefined();
```

Comment:
why not wait for it to get ready?

Reply:
@bipuladh Expansion will be triggered post installation, where all the scenarios will be tested. Installation should check the `Ready` state after ocs cluster creation. Post expansion, Ready check is already being checked.